dfusion_rust_core/src/models/order.rs

```diff
@@ -1,12 +1,18 @@
 use byteorder::{BigEndian, WriteBytesExt};
+use graph::data::store::Entity;
 use serde_derive::{Deserialize};
-use web3::types::H256;
+use std::sync::Arc;
+use web3::types::{H256, U256, Log};
 
 use crate::models::{Serializable, RollingHashable, iter_hash};
 
+use super::util::*;
+
 #[derive(Debug, Clone, Deserialize, Eq, Ord, PartialEq, PartialOrd)]
 #[serde(rename_all = "camelCase")]
 pub struct Order {
+    pub slot: U256,
```

Comment:
Maybe this should be an Optional struct auctionInformation which contains two fields (slot and slotIndex) but can be None for StandingOrders?